packages/dependency-profiles/src/index.js

```diff
@@ -0,0 +1,314 @@
+// This file was generated by 'build.js'
+/* eslint-disable */
+module.exports = {
+  "latest": {
+    "core-win32": {
+      "name": "@office-iss/react-native-win32",
+      "version": ">=0.68.0 <0.68.6"
+    },
+    "core": {
+      "name": "react-native",
+      "version": "^0.68.0"
+    },
+    "core-macos": {
+      "name": "react-native-macos",
+      "version": "^0.68.0"
+    },
+    "core-windows": {
+      "name": "react-native-windows",
+      "version": "^0.68.0"
+    },
```

Comment:
I don't think you want to add these. They overwrite consumers' react-native version. Instead, if you want people to use 0.68, you should rename `latest` to `0.68`. This will tell dep-check to append your list of dependencies to the profile for react-native 0.68.

Reply:
Thanks! I was confused on whether or not to add these. That makes sense. Do you see any issue with just maintaining a `latest` profile instead of a profile for specific react-native versions?

Reply:
Currently, consumers select a profile to align against, e.g. to ensure all your dependencies are compatible with react-native 0.68, one would typically pick the profile named "0.68". The way you extend/override the profile is to have your own custom profile whose name matches the profile you want to extend. This means that if you want to add FURN packages to the 0.68 profile, you need to also name it "0.68". `latest` doesn't exist so it would just become a standalone profile.

Reply:
I see. I've updated this to define profiles at a specific react-native version. Since we don't maintain older releases, I didn't add any logic to maintain older RN profiles. Instead, I added a comment in the `update-profile` script instructing maintainers to "freeze" the old profile to a new file when updating to a newer RN version and append any older profiles to the list.